tests/test_A0_initialization.py

```diff
@@ -8,79 +8,92 @@
 from mvs_eland_tool.mvs_eland_tool import main
 
-from .constants import REPO_PATH
+from .constants import REPO_PATH, CSV_ELEMENTS, INPUTS_COPY
 
-OUTPUT_PATH = os.path.join(".", "tests", "other")
-
-
-class TestOutputPath:
-
-    output_path = os.path.join(".", "tests", "MVS_outputs")
-
-    def setup_method(self):
-
-        os.mkdir(self.output_path)
-        os.mkdir(os.path.join(self.output_path, "dummy"))
-
-    def test_create_output_path_ask_to_overwrite_existing_no(self):
-        with mock.patch("builtins.input", return_value="no"):
-            with pytest.raises(FileExistsError):
-                initializing.check_output_directory(
-                    os.path.join(self.output_path), overwrite=False
-                )
-            assert os.path.exists(self.output_path)
-            assert os.path.exists(os.path.join(self.output_path, "dummy"))
-
-    def test_create_output_path_ask_overwrite_existing_yes(self):
-        with mock.patch("builtins.input", return_value="yes"):
-            initializing.check_output_directory(
-                os.path.join(self.output_path), overwrite=False
-            )
-            assert os.path.exists(self.output_path)
-            assert not os.path.exists(os.path.join(self.output_path, "dummy"))
-
-    def test_create_output_path_existing_yes(self):
-        initializing.check_output_directory(
-            os.path.join(self.output_path), overwrite=True
-        )
-        assert os.path.exists(self.output_path)
-        assert not os.path.exists(os.path.join(self.output_path, "dummy"))
-
-    def teardown_method(self):
-        shutil.rmtree(self.output_path, ignore_errors=True)
 
-
-class TestUserInput:
+class TestProcessUserArguments:
 
     output_path = os.path.join(".", "tests", "MVS_outputs")
     input_path = os.path.join("tests", "inputs")
     fake_input_path = os.path.join("tests", "fake_inputs")
 
-    def test_user_input_path_folder_copy_in_output(self):
-        initializing.get_user_input(path_output_folder=self.output_path)
-        assert os.path.exists(self.output_path)
-
-    def test_user_input_path_input_folder_not_existing(self):
-        with pytest.raises(NotADirectoryError):
-            initializing.get_user_input(
-                path_input_file=self.fake_input_path,
-                path_output_folder=self.output_path,
-            )
-
-    def test_user_input_path_input_file_not_existing(self):
-        with pytest.raises(FileNotFoundError):
-            initializing.get_user_input(
-                path_input_file=os.path.join(self.input_path, "not_existing.json"),
-                path_output_folder=self.output_path,
-                overwrite=True,
-            )
+    def test_input_folder_is_copied_in_output_within_folder_named_input(self):
+        pass
+        # initializing.get_user_input(path_output_folder=self.output_path)
+        # assert os.path.exists(self.output_path)
+
+    def test_input_folder_not_existing_raise_filenotfound_error(self):
```

Comment:
```suggestion
    def test_input_folder_not_existing_raise_foldernotfound_error(self):
```

Reply:
If you defined an errortype "FileNotFound", its fine.

Reply:
Is it builtin error :)